fixtures/ed/ntfs/admin_stations.txt

```diff
@@ -0,0 +1,2 @@
+admin_id,admin_name,station_id,station_name,stop_id,stop_name
+admin:A,"admin A",A,"Arrêt A",SA:A,"Arrêt A"
```

Comment:
You can put different `name`s to check you don't read them (I guess it's the one from stops.txt that's picked):
```suggestion
admin:A,"admin A",A,"Toto A",SA:A,"Tata A"
```
Or maybe just remove the `assert` from the test?

> Same remark for the `ntfs_dst` file

Reply:
Good catch. More clear :tada: